meson/config/meson.build

```diff
@@ -1 +1,2 @@
 add_project_arguments('-DHAVE_CONFIG_H', language: ['c', 'cpp'])
+conf.set_quoted('PDNS_CONFIG_ARGS', meson.build_options(), description: 'Build options')
```

Comment:
This change causes `PDNS_CONFIG_ARGS` to be defined when building dnsdist, which means `logger.hh` is included in `remote_logger.cc` and it doesn't exist :-/

Reply:
Grumble, I was trying to mimick what autotools does to get config details back in `--version`, will revisit.